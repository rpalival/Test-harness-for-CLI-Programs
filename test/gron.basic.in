{
    "hello": "world",
    "number": 123
}