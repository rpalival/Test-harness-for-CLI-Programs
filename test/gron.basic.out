json = {};
json.hello = "world";
json.number = 123;